pages/wiki/useful-commands.md

```diff
@@ -91,8 +93,36 @@ systemctl --force reboot recovery # Restarts the watch in recovery mode
 
 Although we offer [synchronization clients](https://asteroidos.org/wiki/synchronization-clients/) for different platforms, it might still be useful to set the time zone or synchronise the date and time using standard Linux tools. The Linux folder `/usr/share/zoneinfo/` contains the naming scheme for your local time zone in `<continent>/<zone>` format.
 
+<b>Please Note</b> that setting a time zone will break time sync with the forementioned client apps.
+
 ```
 ssh root@192.168.2.15 "timedatectl set-timezone <continent>/<zone>" # e.g. Europe/Berlin
 ssh root@192.168.2.15 "date -s @`(date -u +"%s" )`"
 ```
 
+# Boot to fastboot bootloader menu
+
+---
+
+In case you can not boot to the fastboot bootloader menu using `adb reboot bootloader`, try the manual method for your watch specified here. These methods come in very handy in case your watch does neither boot AsteroidOS, nor WearOS successfully in its current state.
+
+&nbsp;
+#### Anthias, Bass, Dory, Lenok, Mooneye, Sparrow, Swift, Wren
+During first seconds of the boot process, swipe diagonally across the screen. Starting from the top/left outer edge of the screen to the bottom/right.
+
+&nbsp;
+#### Catfish, Skipjack
+Power down the watch. Keep holding both buttons during the boot process until the fastboot menu appears.
+
+&nbsp;
+#### Firefish
+After the watch stops vibrating on startup, immediately touch the top/left & bottom/right of the screen and press the middle button while the manufacturer logo is shown.
```

Comment:
Remove passive voice:
After the watch stops vibrating on startup, immediately touch the top/left & bottom/right of the screen and press the middle button while the manufacturer logo ~is shown~ displays.